docs/fos_http_cache.md

```diff
@@ -0,0 +1,86 @@
+# FOSHttpCacheBundle usage in eZ
+
+Introduced with eZ Publish 5.4, [FOSHttpCacheBundle][fos] is supported by eZ Platform, and covers the following
+features:
+
+* Tags purging/baning (since version 1.8)
```

Comment:
1.9, and maybe mention you are talking about eZ Platform as opposed to Publish above